### DISCUSSION TITLE
Use GPIO for other modules?

### DISCUSSION THREAD

**Discussion by Jumbo125** [rern/rAudio#1850] [General]
Hello everyone,

Thank you very much for this amazing project. As far as I’ve seen, it runs on Arch Linux.
This leaves room for countless other options.

I would like to use some sensors on the Raspberry Pi. For this, I need the I2C protocol, which by default uses GPIO 2 and GPIO 3.
Are these pins occupied by this project, or can I use them freely?
Which GPIOs are already preconfigured and therefore reserved by the project?
Thank you so much!

**Comment by rern** [comment]
Yes, you can use them.

**Comment by Jumbo125** [comment]
okay. thank you!
